benchmark/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/{{ global.project }}/ubuntu:18.04
+FROM {{ global.docker_prefix }}/ubuntu:18.04
```

Comment:
Could this use `DOCKER_ROOT_IMAGE`?

Reply:
Changed to DOCKER_ROOT_IMAGE, plus in a few other places where ubuntu:18.04 is used and it made sense to me.